src/core/execute.c

```diff
@@ -732,21 +732,21 @@ static int ask_for_confirmation(const char *vc, const char *cmdline) {
                 return 1;
         }
 
-        r = ask_char(&c, "yns", "Execute %s? [Yes, No, Skip] ", cmdline);
+        r = ask_char(&c, "yfs", "Execute %s? [Yes, Fail, Skip] ", cmdline);
```

Comment:
well 'n' or "no" is rather ambiguous: does it mean "don't execute but assume the command succeed (aka 's')" ? or does it mean "don't execute but assume the command failed" ?

I actually found myself pressing 'n' a couple of time thinking 'n' as 'next', like the 'next" command available in gdb. And here I expected 'n' to have the same behavior as 'y'.

So quite confusing... therefore I dont think we should plug 'n'.